bug: ESLint errors block build - unused variables in Toolbar.svelte

## Description

Toolbar.svelte has two unused variables that cause ESLint errors and fail the lint check:

- Line 50: `hasSelection` is declared but never used
- Line 62: `ondelete` is declared but never used

## Current Behavior

Running `npm run lint` fails with:
```
src/lib/components/Toolbar.svelte
  50:5  error  'hasSelection' is assigned a value but never used
  62:5  error  'ondelete' is assigned a value but never used
```

**Note:** The Vite build succeeds, but lint fails. This blocks CI pipeline if lint is a required check.

## Expected Behavior

Lint should pass. Either:
1. Remove unused props from the Props interface and destructuring
2. Use these props in the component (if they should have functionality)

## Files Affected

- `src/lib/components/Toolbar.svelte`

## Priority

**High** - Fails lint check in CI/CD pipeline

---

Found during #578 audit.

--- a/src/lib/components/Toolbar.svelte
+++ b/src/lib/components/Toolbar.svelte
@@ -25,7 +25,6 @@
   import { analytics } from "$lib/utils/analytics";
 
   interface Props {
-    hasSelection?: boolean;
     hasRacks?: boolean;
     theme?: "dark" | "light";
     displayMode?: DisplayMode;
@@ -37,7 +36,6 @@
     onload?: () => void;
     onexport?: () => void;
     onshare?: () => void;
-    ondelete?: () => void;
     onfitall?: () => void;
     ontoggletheme?: () => void;
     ontoggledisplaymode?: () => void;
@@ -47,7 +45,6 @@
   }
 
   let {
-    hasSelection = false,
     hasRacks = false,
     theme = "dark",
     displayMode = "label",
@@ -59,7 +56,6 @@
     onload,
     onexport,
     onshare,
-    ondelete,
     onfitall,
     ontoggletheme,
     ontoggledisplaymode,

